MonoGame.Framework/Audio/Xact/XactHelpers.cs

```diff
@@ -53,7 +53,7 @@ public static float ParseVolumeFromDecibels(byte decibles)
         public static float ParseVolumeFromDecibels(float decibles)
         {
             // Convert from decibles to linear volume.
-            return (float)Math.Pow(10.0, decibles / 20.0);
```

Comment:
Precision lost!
Here 'decibles' (decibels?) were upgraded to double to be divided by 20.0.
And then Power was calculated with double precision.